### PR TITLE
feat: remove nodes

### PR DESCRIPTION
**Pull request by Amyh102**
Remove nodes from edit page with delete button
Screen Shot 2022-03-31 at 4 05 50 AM

### CODE REVIEW

**Comment by jodijang** [comment]
Just a couple of comments:

- could you use the IconButton component for the trash button? It'll give the page a more consistent style and on-hover effects
- could you make it so the graph doesn't change size on hover?
- could you add the ConfirmationModal component when the user clicks the trash button? There's an example of using the confirmation modal being used when projects are deleted

File: src/components/wrappers/GraphNodeWrapper/GraphNodeWrapper.module.scss [hunk 1]
```diff
@@ -32,6 +32,23 @@
       font-size: 12px;
     }
   }
+
+  .delete {
+    display: flex;
+    justify-content: end;
```
Comment (jodijang) [inline src/components/wrappers/GraphNodeWrapper/GraphNodeWrapper.module.scss]:
```suggestion
    justify-content: flex-end;
```

**Review by jodijang — COMMENTED** [review]
Could you please modify this such that only the trash button visibility changes on hover on the graph?

Currently, all edit buttons (pencil next to title, description, and graph) visibility changes on hover on *any* editable component.

The other fixes look good!

File: src/components/IconButton/icons.tsx [hunk 2]
```diff
@@ -51,3 +51,15 @@ export const close = (
...
+    <path d="M10 12V16M14 12V16" strokeWidth="1.5" strokeLinecap="round" />
+  </svg>
+)
```
Comment (jodijang) [inline src/components/IconButton/icons.tsx]: I think this icon is redundant with the "remove" icon

File: src/components/wrappers/GraphNodeWrapper/GraphNodeWrapper.module.scss [hunk 3]
```diff
@@ -32,6 +32,23 @@
...
+  }
+
+  button {
```
Comment (jodijang) [inline src/components/wrappers/GraphNodeWrapper/GraphNodeWrapper.module.scss]: Do you still need this now that you're using the IconButton component?

File: src/components/IconButton/icons.tsx (hunk 2 above)
Comment (Amyh102) [reply src/components/IconButton/icons.tsx]: do you want to replace the trash can with the "x" icon?

Comment (jodijang) [reply src/components/IconButton/icons.tsx]: No, the "x" icon is called "close", and the trash icon is called "remove"

**Review by jodijang — APPROVED** [review]

**Pull request merged**